src/pages/bug-bounty/_hero.tsx

```diff
@@ -1,29 +1,59 @@
 import React from 'react'
-import { graphql, useStaticQuery } from 'gatsby'
-import HeroComponent from './components/_hero_component'
+import styled from 'styled-components'
+import { StaticImage } from 'gatsby-plugin-image'
 import { useBrowserResize } from 'components/hooks/use-browser-resize'
+import { Container } from 'components/containers'
+import device from 'themes/device'
+import { BackgroundImageWrapper, StaticImageWrapper, Header } from 'components/elements'
+import { localize } from 'components/localization'
 
-const query = graphql`
-    query {
-        hero_background_desktop: file(
-            relativePath: { eq: "bug-bounty/security-overlay-desktop.jpg" }
-        ) {
-            ...heroImage
-        }
-        hero_background_mobile: file(
-            relativePath: { eq: "bug-bounty/security-overlay-mobile.jpg" }
-        ) {
-            ...heroImage
-        }
+const StyledContainer = styled(Container)`
+    height: 100%;
+    padding: 160px 222px;
+
+    @media ${device.tabletL} {
+        padding: 120px 16px;
+        width: 100%;
     }
 `
 
+const Subheadline = styled(Header)`
+    font-weight: normal;
+    margin-top: 16px;
+`
+
 const Hero = () => {
-    const data = useStaticQuery(query)
     const [is_mobile] = useBrowserResize()
-    const background = is_mobile ? data['hero_background_mobile'] : data['hero_background_desktop']
 
-    return <HeroComponent background_data={background} background_dark="0.3" />
+    return (
+        <BackgroundImageWrapper>
+            <StaticImageWrapper>
+                {is_mobile ? (
+                    <StaticImage
+                        src="../../images/common/bug-bounty/security-overlay-mobile.jpg"
+                        alt="deriv security"
+                        style={{ opacity: '0.3' }}
```

Comment:
```suggestion
                        src="../../images/common/bug-bounty/security-overlay-mobile.jpg"
                        alt="deriv security"
                        style={{ opacity: '0.3' }}
```
Is this image at the top of the page? if yes add eager loading props `loading="eager"`. The same for below

Reply:
Done